### ISSUE TITLE
Deployment fails with frdm_k64f on some systems due to too fast unmounting after binary copying

### ISSUE THREAD

**Issue by pfalcon** [Linaro/lite-lava-docker-compose#3]
`make testjob` from this repo ends with:

~~~
nice mount -t vfat /dev/disk/by-id/usb-MBED_microcontroller_024002269ee94e660000000000000000000000006330b3de-0:0 /tmp/tmpmmvrpiln
Copying /var/lib/lava/dispatcher/tmp/10/deployimages-ufqamsna/zephyr/zephyr.bin to /tmp/tmpmmvrpiln
nice umount /dev/disk/by-id/usb-MBED_microcontroller_024002269ee94e660000000000000000000000006330b3de-0:0
end: 2.1.1 flash-cmsis (duration 00:00:00) [common]
start: 2.1.2 connect-device (timeout 00:01:00) [common]
[common] connect-device Connecting to device using 'telnet ser2net 5001'
end: 2.1.2 connect-device (duration 00:00:00) [common]
end: 2.1 boot-cmsis-retry (duration 00:00:00) [common]
end: 2 boot-cmsis (duration 00:00:00) [common]
start: 3 lava-test-monitor-retry (timeout 00:02:00) [common]
start: 3.1 lava-test-monitor (timeout 00:02:00) [common]
Waiting for start message: (tc_start\(\)|starting .*test|BOOTING ZEPHYR OS)
Trying 172.18.0.3...
Connected to ser2net.
Escape character is '^]'.
wait for prompt timed out
end: 3.1 lava-test-monitor (duration 00:00:30) [common]
~~~

On the boards side, a red LED lights up to ~50% of intensity (this LED isn't usually lit), and USB FS exports file `FAIL.TXT` with contents: `Flash algorithm write command FAILURE`. This happens without any blinking of green LED of noticeable duration, which normally accompanies flashing process (i.e. when host copies a file to device's USB FS).

Pressing RESET button on the board doesn't clear error condition (LED/file). Disconnecting USB cable is required.

**Comment by pfalcon** [comment]
Copying a file manually on the host leads to normal deployment (with green LED blinking for a few seconds as flashing goes on, then deployed binary runs successfully after reset).

**Comment by pfalcon** [comment]
This is with:

~~~
Version: 0226
Build:   Aug 24 2015 17:06:30
Git Commit SHA: 27a236b9fe39c674a703c5c89655fbd26b8e27e1
Git Local mods: Yes
~~~

Toggling `resets_after_flash = false` to true doesn't help (this board with this daplink version requires it to be false normally).

All this setup works ok on Thinkpad X230.

**Comment by erwango** [comment]
Facing some issue on my side.

**Comment by pfalcon** [comment]
My patch to fix this issue, https://git.lavasoftware.org/lava/lava/merge_requests/620, was merged to LAVA master, should be in LAVA 2019.07.

**Comment by pfalcon** [comment]
This is now in released LAVA.
